samples/net/echo_server/src/echo-server.c

```diff
@@ -6,11 +6,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#if 1
-#define SYS_LOG_DOMAIN "echo-server"
-#define NET_SYS_LOG_LEVEL SYS_LOG_LEVEL_DEBUG
-#define NET_LOG_ENABLED 1
-#endif
+#define LOG_MODULE_NAME echo_server
+#define NET_LOG_LEVEL 4
```

Comment:
Why switch to a hard coded 4 here?

Reply:
> Why switch to a hard coded 4 here?

I guess it is one-to-one translation. Was:

> #define NET_SYS_LOG_LEVEL SYS_LOG_LEVEL_DEBUG

now:

> #define NET_LOG_LEVEL 4

Except, as mentioned by @pizi-nordic, should used symbolic constants everywhere.